x-pack/plugins/security_solution/public/timelines/components/timeline/tabs_content/index.tsx

```diff
@@ -215,7 +216,7 @@ const ActiveTimelineTab = memo<ActiveTimelineTabProps>(
         >
           {isGraphOrNotesTabs && getTab(activeTimelineTab)}
         </HideShowContainer>
-        {hasAssistantPrivilege ? getAssistantTab() : null}
+        {hasAssistantPrivilege && !aiAssistantFlyoutMode ? getAssistantTab() : null}
```

Comment:
Now that we have the `AI Assistant` header button which makes it always visible (even when timeline is open), I think we can go ahead and remove the tab regardless of if the flyout feature flag is on. 🙂 Yay cleanup!